src/components/RenderHTML.js

```diff
@@ -114,7 +116,7 @@ function ImgRenderer({tnode}) {
 
     return (
         <AttachmentModal
-            title="Attachment"
+            title={translate(ONYXKEYS.PREFERRED_LOCALE, 'attachment')}
```

Comment:
Why is this not using withLocalize?

Reply:
I am unable to pass translation prop from [RenderHTML](https://github.com/eVoloshchak/Expensify.cash/blob/fe1aad8926b9bd80d8d63bbcc18e6e449d3e26c0/src/components/RenderHTML.js#L156) to [ImgRenderer](https://github.com/eVoloshchak/Expensify.cash/blob/fe1aad8926b9bd80d8d63bbcc18e6e449d3e26c0/src/components/RenderHTML.js#L81)

Can you please point me in the right direction?

Reply:
Not sure I follow why you would need to pass it from RenderHTML to ImgRenderer...